packages/starlight/index.ts

```diff
@@ -26,6 +27,7 @@ export default function StarlightIntegration(opts: StarlightUserConfig): AstroIn
 		name: '@astrojs/starlight',
 		hooks: {
 			'astro:config:setup': ({ config, injectRoute, updateConfig }) => {
+				const useTranslations = createTranslationSystemFromFs(userConfig, config);
```

Comment:
This variable is not in use yet, but would be passed into anything that needs it, such as the Starlight integration added in #742.